enricher/fabric8/src/main/resources/META-INF/fabric8/enricher-default

```diff
@@ -5,3 +5,4 @@ io.fabric8.maven.enricher.fabric8.IconEnricher
 io.fabric8.maven.enricher.fabric8.ExposeEnricher,1000
 io.fabric8.maven.enricher.fabric8.SpringBootHealthCheckEnricher
 io.fabric8.maven.enricher.fabric8.KarafHealthCheckEnricher
+io.fabric8.maven.enricher.fabric8.PrometheusEnricher
```

Comment:
While we are at it, I don't know whether an important bug fix made it already to branch `-redhat-80` which caused enrichers with the same order to overwrite each other. (I will check that later and send an appropriate PR in case).

However, I suggest to use orders on _all_ enrichers since the enrichers from the modules _fabric8_ and _standard_ are merged into one list and if no order are given explicitely the order is not really deterministic (it depends which jar is found first in the classpath* lookup). Again an argument for #709 which lets define an overall order for all enricher directly without resorting to order number and conventions across module (which again introduces implicit and hard to detect dependencies).